crates/plugins/mediator-coordination/src/didcomm/bridge.rs

```diff
@@ -36,25 +37,37 @@ impl DIDResolver for LocalDIDResolver {
             return Ok(Some(self.diddoc.clone()));
         }
 
-        if !did.starts_with("did:key:") {
+        if did.starts_with("did:peer:") {
+            // Adding logic to resolve Peer DIDs
+            let method = DidPeer::new_with_format(PublicKeyFormat::Jwk); // Assuming a Peer DID resolver
+            match method.expand(did) {
+                Ok(diddoc) => Ok(Some(
+                    serde_json::from_value(json!(Document {
+                        service: Some(vec![]),
+                        ..diddoc
+                    }))
+                    .expect("Should easily convert between DID document representations."),
+                )),
```

Comment:
What @Hermann-Core is saying is that you should propagate the error as such or otherwise to avoid panics

Reply:
Okay i get it :+1: